postgres/server/init.sls

```diff
@@ -261,12 +261,21 @@ postgresql-tablespace-dir-{{ name }}-fcontext:
 
 {%- if not postgres.bake_image %}
 
+# Workaround for FreeBSD minion undefinitely hanging on service start
+# cf. https://github.com/saltstack/salt/issues/44848
+{% if postgres.service.sysrc %}
+posgresql-rc-flags:
+  sysrc.managed:
+    - name: {{ postgres.service.name }}_flags
+    - value: "{{ postgres.service.flags }} > /dev/null 2>&1"
```

Comment:
@sticky-note Please apply this patch here:

```diff
diff --git a/postgres/server/init.sls b/postgres/server/init.sls
index 21b33a6..ce1d575 100644
--- a/postgres/server/init.sls
+++ b/postgres/server/init.sls
@@ -268,6 +268,8 @@ posgresql-rc-flags:
   sysrc.managed:
     - name: {{ postgres.service.name }}_flags
     - value: "{{ postgres.service.flags }} > /dev/null 2>&1"
+    - watch_in:
+      - service: postgresql-running
 {% endif %}
 
 # Start PostgreSQL server using OS init
```

Reply:
Oh yes, you're right. Well seen